doc/src/build/fullnode.md

```diff
@@ -288,3 +288,119 @@ If you followed the instructions for [Building from Source](#building-from-sourc
+  # When this is 1, pruner will only prune object versions from transaction checkpoints
+  # older than the current epoch. In general, when this is N (where N >= 1), pruner 
+  # will only prune object versions from checkpoints upto current - N epoch. 
+  # This means it is possible to have multiple versions of an object to be present 
+  # in the db. It is a recommended setting for fullnodes as they may have to serve 
+  # rpc requests which require looking up objects by id and version (instead of just latest
+  # version). However, if a fullnode is not serving such rpc requests it should also 
+  # use aggressive pruning
+  num-epochs-to-retain: 0
+  # Advanced setting: Maximum number of checkpoints in one batch of pruning run. This is
+  # mostly not needed to be changed and default is good for most use cases
+  max-checkpoints-in-batch: 10
+  # Advanced setting: Maximum number of transactions in one batch of pruning run. This is
+  # mostly not needed to be changed and default is good for most use cases
+  max-transactions-in-batch: 1000
+```
+## Transaction Pruning
+
+Transaction pruning is another type of pruning that removes old transactions and effects from the db. 
+Checkpoints which are created periodically contain transaction and their effects. 
+Transaction pruning works in the background by tailing checkpoint stream as well.
+Full nodes and validators can enable transaction pruning by adding can enable transaction pruning
+by adding `num_epochs_to_retain_for_checkpoints` to their `authority-store-pruning-config` config:
+
+```yaml
+authority-store-pruning-config:
+  num-latest-epoch-dbs-to-retain: 3
+  epoch-db-pruning-period-secs: 3600
+  num-epochs-to-retain: 0
+  max-checkpoints-in-batch: 10
+  max-transactions-in-batch: 1000
+  # Number of latest epochs before which transaction and effects could be pruned.
+  # When this is N (where N >= 2), pruner can prune transaction and effects from 
+  # checkpoints upto current - N epoch. Transaction and effects from current and
+  # previous epoch are never pruned. N = 2 is a recommended setting for validators
+  # and fullnodes not serving rpc requests.
+  num_epochs_to_retain_for_checkpoints: 2
+  # Ensures that individual SST files periodically go through the compaction process.
+  # This helps reclaim disk space and avoid fragmentation issues
+  periodic-compaction-threshold-days: 1
+```
+
+## Archival Fallback
+
+Once various full nodes start pruning historical transactions and effects, it may not be always possible for
+their peers to catch up transaction and effects via state synchronization. And for this a node needs to fallback
+downloading this data from a remote aws s3 bucket. This archive has all transaction and effects data since 
+genesis and lags behind the latest network checkpoint by ten minutes. Enabling this is very highly recommended
+for all nodes. A sui node can enable this automatic archive fallback by adding the following config:
```

Comment:
After Sui starts performing transaction pruning on Full nodes to remove historical transactions and their effects, it might not be possible for peer nodes to catch up with the transactions and effects via synchronization. Instead, peer nodes can fall back by downloading and restoring a [database snapshot](../build/snapshot.md). 
The snapshot is an archive of all transaction data on Sui, trailing behind the latest checkpoint by 10 minutes. You should enable this for all nodes as a best practice. To configure a Sui node to automatically fall back to a snapshot image, add the following to your config file.

What is the name and location of this file, the fullnode.yaml?

Reply:
I don't want to use the word snapshot here because that is a slightly different concept. But used all the other parts of your suggested change